src/tests/sortingTests.spec.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixture';
+import { SORTING_OPTIONS_FUNCTIONS } from '../pages/Inventory.page';
+
+test.describe('Perform and verify sorting', () => {
+    test.beforeEach(async ({ loginPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+    });
+
+    test('Verify sorting by Name(A-Z)', async ({ inventoryPage }) => {
```

Comment:
Read this article to make one test from the below 4.
https://playwright.dev/docs/test-parameterize

The difference is only in `sortingOption`. So you can combine 4 tests into 1 while in report it will display 4 tests